platform-service-framework/src/main/java/org/hypertrace/core/serviceframework/jvm/JVMDiagnosticCommand.java

```diff
@@ -59,6 +59,7 @@ public String invoke(final Op operation, final String[] args) {
    */
   public enum Op {
     JVM_THREADS("threadPrint", "Thread.Print"),
+    JVM_THREADS_DUMP_TO_FILE("threadDumpToFile", "Thread.dump_to_file", "/tmp/jvm_thread_dump"),
```

Comment:
virtual threads will not be shown in Thread.Print, only will be shown in dump to file